### ISSUE TITLE
Paid transfer gets stuck v1.13.2

### ISSUE THREAD

**Issue by Angelo-gh3990** [filecoin-project/lotus#8009]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [X] lotus miner/market - retrieval deal
- [X] lotus miner/market - data transfer
- [X] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
Daemon:  1.13.2+mainnet+git.f9db71ee1+api1.2.0
Local: lotus-miner version 1.13.2+mainnet+git.f9db71ee1
```

### Describe the Bug

When I try to retrieve data from my miner through the lotus client it stops working after a while, command being `used` :

`lotus client retrieve --from "wallet address" --maxPrice 1 --provider f0xxxxx mDGHg5AIgzmWXB-P4hE8NojRdaMK/lGivooVtNzA4CabNRipbnu0 --car`

### Logging Information

Miner :

set price:
lotus-miner retrieval-deals set-ask --price 0.00000001 --unseal-price 0.0000001 --payment-interval 1073741824 --payment-interval-increase 1073741824

check price:

lotus-miner retrieval-deals get-ask
Price per Byte            Unseal Price   Payment Interval  Payment Interval Increase
0.000000000000000009 FIL  0.0000001 FIL  1GiB              1GiB

logging lotus node:

2022-01-28T05:51:37.421Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.431Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.441Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.452Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.463Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.473Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.483Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.494Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.504Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.515Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.524Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.536Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.544Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.553Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.565Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.573Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.583Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.594Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.603Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.612Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.620Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.629Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.641Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.652Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.663Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.673Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.684Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.697Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.708Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.720Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.729Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.740Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.751Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}
2022-01-28T05:51:37.764Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusAccepted", "message": ""}

2022-01-28T05:56:15.419Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.466Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.509Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.553Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.603Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.649Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.703Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.757Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.816Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.862Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.905Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.951Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:15.995Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.041Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.095Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.138Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.183Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.224Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.267Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.332Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.389Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.439Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.482Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.527Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.570Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.612Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.655Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.699Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.761Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.813Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.856Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.901Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.943Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:16.987Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:17.030Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:17.073Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}
2022-01-28T05:56:17.138Z        INFO    markets loggers/loggers.go:25   retrieval client event  {"name": "ClientEventBlocksReceived", "deal ID": "1643252648484489546", "state": "DealStatusFundsNeeded", "message": ""}

lotus client logging:

Recv 1006 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.792s
Recv 1007 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.802s
Recv 1008 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.813s
Recv 1009 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.824s
Recv 1010 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.834s
Recv 1011 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.847s
Recv 1012 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.859s
Recv 1013 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.871s
Recv 1014 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.88s
Recv 1015 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.891s
Recv 1016 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.901s
Recv 1017 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.915s
Recv 1018 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.927s
Recv 1019 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.937s
Recv 1020 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.948s
Recv 1021 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.959s
Recv 1022 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.969s
Recv 1023 MiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.98s
Recv 1 GiB, Paid 0 FIL, BlocksReceived (Accepted), 1m31.991s
Recv 1 GiB, Paid 0 FIL, PaymentChannelCreateInitiated (PaymentChannelCreating), 1m32.001s     <=== PAUZED 2min

Recv 2.21 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.584s
Recv 2.211 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.62s
Recv 2.212 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.658s
Recv 2.213 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.7s
Recv 2.214 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.738s
Recv 2.215 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.779s
Recv 2.216 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.825s
Recv 2.217 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.868s
Recv 2.218 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.915s
Recv 2.219 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.962s
Recv 2.22 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m36.998s
Recv 2.221 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.041s
Recv 2.222 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.098s
Recv 2.223 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.14s
Recv 2.224 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.188s
Recv 2.225 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.229s
Recv 2.226 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.27s
Recv 2.227 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.305s
Recv 2.228 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.339s
Recv 2.229 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.38s
Recv 2.23 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.423s
Recv 2.231 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.467s
Recv 2.232 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.511s
Recv 2.233 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m37.961s
Recv 2.234 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m38.565s
Recv 2.235 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 5m39.306s     <=== PAUZED 2min

Recv 2.99 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.914s
Recv 2.991 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.914s
Recv 2.992 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.915s
Recv 2.993 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.916s
Recv 2.994 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.917s
Recv 2.995 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.918s
Recv 2.996 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.918s
Recv 2.997 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.92s
Recv 2.998 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.921s
Recv 2.999 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.921s
Recv 3 GiB, Paid 0.000000009664165926 FIL, BlocksReceived (FundsNeeded), 6m30.922s
Recv 3 GiB, Paid 0.000000009664165926 FIL, SendFunds (SendFunds), 6m30.922s
Recv 3 GiB, Paid 0.00000002899247742 FIL, PaymentSent (Ongoing), 6m30.982s
Recv 3 GiB, Paid 0.00000002899247742 FIL, PaymentRequested (FundsNeeded), 6m31.041s
Recv 3 GiB, Paid 0.00000002899247742 FIL, PaymentRequested (FundsNeeded), 6m31.042s     <=== STOPPED

After 3GB (data is 20GB), it time's out. Retrieving it without a price set through Filc works fine (0 FIL)
The download speed is also a lot slower as it constantly checks the payment channel through lotus.

Any idea's are welcome?

**Comment by jennijuju** [comment]
this should be resolved by #7883 , which will be included in v1.15.1

**Comment by rjan90** [comment]
Hey!

Thanks for the report. This issue has been resolved in a upcoming release v1.15.1. If you still see this issue on that branch, please open a new ticket.

Closing ticket - thanks again!
